app/src/main/java/com/gpayinterns/chat/LoginActivity.java

```diff
@@ -51,6 +51,8 @@
     private static final String TAG = "SignInActivity";
     private static final String EMAIL = "email";
 
+    private boolean active;
```

Comment:
What does this field represent?
Please add a documentation comment.

Reply:
okay sure